parser.y

```diff
@@ -1133,6 +1135,17 @@ AlterTableStmt:
 			Specs: specs,
 		}
 	}
+|	"ALTER" IgnoreOptional "TABLE" TableName AlterTableSpecListOpt "REMOVE" "PARTITIONING"
```

Comment:
Could you make this an `AlterTableSpec` rule on lines 1164–1392 instead?

Reply:
@kennytm `REMOVE PARTITIONING` can't repeat in MySQL.

Reply:
I think `remove partitioning` should not be a kind of `AlterTableSpec`, for the reason that `Restore()` will insert a comma  before it.

Reply:
🤔

Reply:
see:https://github.com/mysql/mysql-server/blob/8.0/sql/sql_yacc.yy#L7900

we can add a rule named `alter_table_partition_options`
```
alter_table_partition_options:
          partition_clause
          {
          }
        | REMOVE_SYM PARTITIONING_SYM
          {
          }
        ;
```

Reply:
@leoppro Sounds good. Let me try it.